components/common/src/command/package/install.rs

```diff
@@ -663,14 +665,8 @@ impl<'a> InstallTask<'a> {
                    ident);
         } else if self.is_offline() {
             return Err(Error::OfflineArtifactNotFound(ident.as_ref().clone()));
-        } else if let Err(err) =
-            retry::retry_future!(delay::Fixed::from(RETRY_WAIT).take(RETRIES),
```

Comment:
We lose this retry logic, correct? Are we ok with that here?

Reply:
It still has the retry logic, I just moved it down to the `fetch_artifact` function that actually calls the Builder API, on line 842 of this file.